### PR TITLE
Allow Downloads From Embeds

### PR DESCRIPTION
**Pull request by gemathus**
Allow file downloads in Iframes. In  #3439 @Limezy introduced tldraw embeds, but downloading files was not working. The `allow-downloads` flag in the frame's sandbox attribute fixes this.

@tommoor not sure why there seems to be a tracked change on the en_US translation. Nothing was changed here, so no worries.

### CODE REVIEW

**Comment by tommoor** [comment]
> not sure why there seems to be a tracked change on the en_US translation

One of the recent merges didn't include it, but it should have – it's no problem.

**Review by tommoor — APPROVED** [review]

**Comment by tommoor** [comment]
Are you able to merge once approved? I'm not sure.

**Comment by gemathus** [comment]
> Are you able to merge once approved? I'm not sure.

I'm not able to merge. I do not have write access.

**Pull request merged**
